gbbs/benchmarks/BatchDynamicConnectivity/SkipList/SkipList.h

```diff
@@ -18,7 +18,10 @@ struct SkipList {
 
         using pointers = std::pair<SkipListElement*, SkipListElement*>;
         using height_array = sequence<pointers>;
-        using values_array = sequence<sequence<sequence<std::pair<uintE, uintE>>>>;
+        using edge_type = std::pair<uintE, uintE>;
+        using values_one_side = sequence<edge_type>;
```

Comment:
```suggestion
        using values_one_side = parlay::sequence<edge_type>;
```
It is better to use the full namespace description. It is more clear where that type is coming from and it avoid collisions if you include another header that defines a type with the same name. (Generally, avoid "using namespace" especially in header files.)